src/commcare_cloud/fab/utils.py

```diff
@@ -202,9 +199,9 @@ def _get_env_filename():
 
 
 def cache_deploy_state(command_index):
-    with open(os.path.join(PROJECT_ROOT, _get_checkpoint_filename()), 'w') as f:
+    with open(os.path.join(PROJECT_ROOT, _get_checkpoint_filename()), 'wb') as f:
```

Comment:
Applies to all of the instances where pickle is used, but from what I read it seems that pickling is always dealing with a byte stream which I why I chose to explicitly have a byte stream on both 2 and 3.